site/en/docs/webstore/branding/index.md

```diff
@@ -107,18 +108,23 @@ services. For further details, check out the [Google Permissions][google-permiss
 
 ## Giving proper attribution {: #giving-proper-attribution}
 
-Do give proper attribution to Google for any use of their [trademarks][google-trademark-list] in
-your extension title or in your description. Example:
+Give proper attribution for the use of any Google trademarks in your extension
+title or in your description. (We publish a [list of Google
+trademarks][google-trademark-list].)
+
+For example, if your item uses the extension title “Highlight local shops for
+Google Maps™”, then you should include the attribution shown below in the item
+description.  Note that the ™ symbol is not needed in the attribution since the
+text itself specifies that the mark (e.g. Google Maps) is a trademark owned by
```

Comment:
The [style guide recommended not using e.g.](https://developers.google.com/style/abbreviations#dont-use).

Reply:
I should have caught that. Someone else's language, so I'm not going try to change at hour 11.